Automattic/jetpack
# Whitelist `jetpack_memberships_site_subscriber` post meta

## Proposed changes:

Sync `jetpack_memberships_site_subscriber` post meta to WPCOM as we rely on this post meta for business logic

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Create a newsletter plan on a JP site that has a JP connection
* Verify that the post and `jetpack_memberships_site_subscriber` post meta is synced to WPCOM
